moto/glue/utils.py

```diff
@@ -90,6 +105,59 @@ def _escape_regex(pattern: str) -> str:
     return pattern.translate(_special_chars_map)
 
 
+def validate_registry_name_pattern_and_length(param_value):
+    param_name = "registryName"
+    max_name_length = MAX_REGISTRY_NAME_LENGTH
+    pattern = RESOURCE_NAME_PATTERN
+    validate_param_pattern_and_length(param_value, param_name, max_name_length, pattern)
+
+
+def validate_arn_pattern_and_length(param_value):
+    param_name = "registryArn"
+    max_name_length = MAX_ARN_LENGTH
+    pattern = ARN_PATTERN
+    validate_param_pattern_and_length(param_value, param_name, max_name_length, pattern)
+
+
+def validate_description_pattern_and_length(param_value):
+    param_name = "description"
+    max_name_length = MAX_DESCRIPTION_LENGTH
+    pattern = DESCRIPTION_PATTERN
+    validate_param_pattern_and_length(param_value, param_name, max_name_length, pattern)
+
+
+def validate_schema_name_pattern_and_length(param_value):
+    param_name = "schemaName"
+    max_name_length = MAX_SCHEMA_NAME_LENGTH
+    pattern = RESOURCE_NAME_PATTERN
+    validate_param_pattern_and_length(param_value, param_name, max_name_length, pattern)
+
+
+def validate_param_pattern_and_length(
+    param_value, param_name, max_name_length, pattern
+):
+    if param_value == "" or len(param_value.encode("utf-8")) > max_name_length:
```

Comment:
Should we check for blanks?

Reply:
I realized the blank check will not be required. Thus, removing blank check.